.github/workflows/pr-integration.yml

```diff
@@ -315,14 +317,14 @@ jobs:
         uses: "./.github/template/deploy-test-prerequisites"
 
       - name: Run tests
-        if: ${{ !matrix.labels.experimental }}
+        if: ${{ !matrix.labels.mode == 'regular' }}
         run: |
           go test -v -timeout=20m ./test/e2e-migrated/logs/... -- -labels=${{ matrix.labels.name }}
 
       - name: Run tests experimental
-        if: ${{ matrix.labels.experimental }}
+        if: ${{ matrix.labels.mode == 'experimental' }}
         run: |
-          go test -v -timeout=20m ./test/e2e-migrated/logs/... -- -labels=${{ matrix.labels.name }},experimental
+          go test -v -timeout=20m ./test/e2e-migrated/logs/... -- -labels=${{ matrix.labels.name }}
```

Comment:
You can remove the whole block. Now the both run test blocks are identical

Reply:
We cannot remove the block since we want to exdecute different set of tests for experimental and release

Reply:
You are executing exactly the same set of tests after your change made in this PR
```
go test -v -timeout=20m ./test/e2e-migrated/logs/... -- -labels=${{ matrix.labels.name }}
```

Reply:
Or am I missing something?

Reply:
fixed